File: src/main/javascript/app/model/EventHistoryItem.js
Ext.define('Spm.model.EventHistoryItem', {
    extend: 'Ext.data.Model',
    alias: 'model.eventHistoryItem',

    fields: [
        {
            name: 'eventType'
        },
        {
            name: 'note'
        },
        {
            name: 'createdDate',
            type: 'date',
            dateFormat: 'd/m/Y H:i'

        },
        {
            name: 'createdBy'
        }
    ]
});

File: bango-web/src/main/java/sonique/bango/domain/EventHistoryItem.java
package sonique.bango.domain;

import java.util.Date;

public class EventHistoryItem {

    private String eventType;
    private String note;
    private Date createdDate;
    private String createdBy;

    public EventHistoryItem(String eventType, String note, Date createdDate, String createdBy) {
        this.eventType = eventType;
        this.note = note;
        this.createdDate = createdDate;
        this.createdBy = createdBy;
    }

    public String eventType() {
        return eventType;
    }

    public String note() {
        return note;
    }

    public Date createdDate() {
        return createdDate;
    }

    public String createdBy() {
        return createdBy;
    }
}
